src/components/course/uc/deleteUserData.unit.test.js

```diff
@@ -0,0 +1,192 @@
+const chai = require('chai');
```

Comment:
It is fine, but it is not a unit test - it is an integration test. We would need to think about it. One quite controversial question - Do we need unit tests?